strapi/strapi
# feat(content-manager): better handle entry duplication

### What does it do?

- No longer fails auto-clone for all relations, but only for bidirectional one-to-one and one-to-many relations
- Displays the problematic fields to the user in a modal in the admin. Uses breadcrumbs when the problematic field is in a component to help the user find it. If the problematic field is in a dynamic zone, show the display name of the component in the breadcrumbs. 

### Why is it needed?

- we were preventing entry duplication in cases where it was possible without danger
- it wasn't clear to the users what was going on, because the error message was vague and complex, and they didn't have time to read it

### How to test it?

- Duplicate entries that have unique fields, and various types of relations. Also test them inside of components and dynamic zones
- When clicking create, you should be redirected to the prefilled edit view for the locale you were looking at
- [Check the designs
](https://www.figma.com/file/pdR8u6hM30yxQUdhvdZWqE/Duplicate?type=design&node-id=1-7075&mode=design&t=l8jdDqUxv2GboADo-0)

### Related issue(s)/PR(s)

fixes #17399
